pkg/flashdb/include/fal_cfg.h

```diff
@@ -26,11 +26,35 @@
 #define FAL_CFG_H
 
 #include "board.h"
+#include "macros/units.h"
+#include "mtd_default.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#if !defined(CONFIG_FLASHDB_MIN_SECTOR_SIZE_DEFAULT_FACTOR) || defined(DOXYGEN)
+/**
+ * @brief   Virtual sector size factor in multiples of 1KiB for FlashDB
+ *
+ * The default is 4 (4096).
+ * FlashDB sector size is different from MTD sector size as it is a
+ * virtual measure of granularity and not a device property.
+ * The virtual sector size must be a multiple of the physical sector size.
+ *
+ * From the documentation of FLashDB:
+ * The internal storage structure of FlashDB is composed of N sectors, and each formatting takes
+ * sector as the smallest unit. A sector is usually N times the size of the Flash block.
+ * For example, the block size of Nor Flash is generally 4096.
+ *
+ * By default, KVDB will use 1 times the block size as the sector size, that is, 4096.
+ * At this time, the KVDB cannot store a KV longer than 4096. If you want to save, for example,
+ * a KV with a length of 10K, you can use the control function to set the sector size to 12K or larger.
+ *
+ */
+#define CONFIG_FLASHDB_MIN_SECTOR_SIZE_DEFAULT_FACTOR   4
```

Comment:
```suggestion
#define CONFIG_FLASHDB_MIN_SECTOR_SIZE_DEFAULT_KiB      4
```

Instead of `FACTOR` you can just use the unit.

The MIN DEFAULT is a bit confusion, maybe add a note that this is only for the automatic partition configuration.